lib/screens/release_health/release_card.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_echarts/flutter_echarts.dart';
+import 'package:sentry_mobile/types/project.dart';
+import 'package:sentry_mobile/types/release.dart';
+
+class ReleaseCard extends StatelessWidget {
+  ReleaseCard({@required this.project, @required this.release});
```

Comment:
as they are only 2 and different data type, it'd be fine to not be named params, but if the number of params is gonna grow, all for it